templates/python/custom-copilot-rag-customize/src/local_data_source.py

```diff
@@ -0,0 +1,66 @@
+import os
+from dataclasses import dataclass
+
+from teams.ai.tokenizers import Tokenizer
+from teams.ai.data_sources import DataSource
+from teams.state.state import TurnContext
+from teams.state.memory import Memory
+
+@dataclass
+class Result:
+    def __init__(self, output, length, too_long):
+        self.output = output
+        self.length = length
+        self.too_long = too_long
+
+class LocalAISearchDataSource(DataSource):
```

Comment:
Also update the class name if we don't want to include "AI Search" in the data source name.

Reply:
done